File: Server/BLL/Models/OrderModel.cs
using BLL.Enum;

namespace BLL.Models;

public class OrderModel
{
    public Guid? Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Email { get; set; }
    public string Phone { get; set; }
    public PaymentType PaymentType { get; set; }
    public DateTime? OrderDate { get; set; } 
    public string UserId { get; set; }
    public string? Comment { get; set; }
}

File: Client/game-store-frontend/src/app/models/order.model.ts
import {PaymentType} from "./payment-type.enum";
import {IGameWithIdInOrderModel} from "./game-with-id-in-order.model";

export interface IOrderModel {
  id?: string,
  firstName: string,
  lastName: string,
  email: string,
  phone: string,
  paymentType: PaymentType,
  orderDate: Date,
  userId?: string,
  comment: string,
  orderedGames: IGameWithIdInOrderModel[]
}

